pkg/csi/service/wcp/controller.go

```diff
@@ -1485,52 +1484,6 @@ func (c *controller) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshot
 				"cns query volume did not return the volume: %s", volumeID)
 		}
 		snapshotSizeInMB := cnsVolumeDetailsMap[volumeID].SizeInMB
-		datastoreUrl := cnsVolumeDetailsMap[volumeID].DatastoreUrl
```

Comment:
Please add a TODO to explain that we may need to add logic to check the max number of snapshots in the future.

Reply:
done